tests/test_models/nn/test_tft.py

```diff
@@ -187,3 +187,11 @@ def test_save_load(example_tsds):
     transform = _get_default_transform(horizon)
     transforms = [transform]
     assert_model_equals_loaded_original(model=model, ts=example_tsds, transforms=transforms, horizon=horizon)
+
+
+def test_repr():
```

Comment:
Don't we want to add the same test for deep-ar? Because it can be strange that we have this test for tft, but doesn't have for deep-ar.

Reply:
I don't think so.
I've added this test because there was issue with TFT.

Otherwise we should add such tests for every representable object not only deepar - and I think it doesn't make much sense for now